tests/tests/es7.math.fscale.ls

```diff
@@ -0,0 +1,14 @@
+{module, test} = QUnit
+module \ES7
+
+test 'Math.fscale' (assert)!->
+  {fscale} = Math
+  assert.isFunction fscale
+  assert.name fscale, \fscale
+  assert.arity fscale, 5
+  assert.looksNative fscale
+  assert.nonEnumerable Math, \fscale
+  assert.same fscale(3 1 2 1 2), 3
+  assert.same fscale(0 3 5 8 10), 5
+  assert.same fscale(1 1 1 1 1), NaN
+  assert.same fscale(-1 -1 -1 -1 -1 ), NaN
```

Comment:
Done. See below. -- I was sweating bullets looking for it. *Never forget to look down...* :smiley: